commit 3e39109b0ec9c4b95a8b737b1dca444ef1938d3c
Author: Jared Casper <jaredcasper@gmail.com>
Date:   Thu Dec 11 21:27:46 2008 -0800

    Compatibility with gcc-4.3
    
    Using auto_ptr in elab_net.cc (added in commit 15702f1, "Fix bad
    memory reference." on 2008-11-28) requires #include <memory> when
    compiling with gcc-4.3.

diff --git a/elab_net.cc b/elab_net.cc
index 96a2cbb..a2ea5a4 100644
--- a/elab_net.cc
+++ b/elab_net.cc
@@ -29,6 +29,7 @@
 
 # include  <iostream>
 # include  <cstring>
+# include  <memory>
 
 /*
  * This is a state flag that determines whether an elaborate_net must
